packages/rum/src/domain/record/mutationObserver.spec.ts

```diff
@@ -431,6 +452,94 @@ describe('startMutationCollection', () => {
         ],
       })
     })
+
+    describe('for shadow DOM', () => {
+      it('should call shadowDomCreatedCallback when host is added', () => {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏
```suggestion
      it('should call addShadowRoot when host is added', () => {
```